docs/modules/ROOT/pages/explanation/organizations-and-rbac.adoc

```diff
@@ -20,3 +29,24 @@ Membership of organizations and organizations itself are currently being maintai
 
 It's planned to enable self-management of organizations and membership in the https://roadmap.appuio.cloud[upcoming Webportal].
 Please xref:contact[contact] us to manage organization membership for you for the time being.
+
+== Transfer Projects
+
+There is the option to move a project from one organization to another.
+This can be done by setting the label `appuio.io/organization` on the `Namespace` of the project to the name of the target organization.
+This needs to be done by someone in the target organization with the permission to edit the `Namespace`.
+
+.As an example: 
```

Comment:
To me this reads quite complicated and represents the advanced case. The simple case is (assuming default RoleBindings): user is member of both source and target organization.